docs/user/generated_config.adoc

```diff
@@ -234,6 +234,10 @@ each of them, with the working directory being the workspace root
 by changing `#rust-analyzer.check.invocationStrategy#` and
 `#rust-analyzer.check.invocationLocation#`.
 
+If `$saved_file` is part of the command, rust-analyzer will pass
+the absolute path of the saved file to the provided command. This is
+intended to be used with non-Cargo build systems.
```

Comment:
Given my comment, let's make clear that this is only a temporary thing and will likely be removed in the future